tests/test_e2e_40_sdntrace.py

```diff
@@ -376,3 +376,23 @@ def test_020_run_sdntrace_fail_missing_flow(self):
             (step['dpid'], step['port']) for step in result[1:-1]
         ]
         assert expected == actual, f"Expected {expected}. Actual: {actual}"
+
+    def test_030_run_sdntrace_for_stored_flows(self):
```

Comment:
This test case is relatively very similar to what `test_001_run_sdntrace_cp` covers. Maybe you could cover the `traces/` endpoint here instead to increase coverage? Covering with 2 or more traces the `traces/` endpoint would be great to have.